Recolor.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class to recolour a transparent image preserving transparency
+ *
+ * @author Robin Andrews (prandrews@hotmail.co.uk)
+ * @author David Yell (neon1024@gmail.com)
+ * @license MIT
+ */
+class Recolor {
+
+    /**
+     * Where the image stored
+     *
+     * @var string
+     */
+    private $imageFolder = __DIR__ . DIRECTORY_SEPARATOR . 'images' . DIRECTORY_SEPARATOR;
```

Comment:
Private is the class properties visibility keyword. This means that no other class, not even a child class can see or access this class property.